modules/core/validator.js

```diff
@@ -262,17 +261,47 @@ export function coreValidator(context) {
   //   `issue` - the issue to focus on
   //
   validator.focusIssue = (issue) => {
-    const extent = issue.extent(context.graph());
-    if (!extent) return;
-
-    const setZoom = Math.max(context.map().zoom(), 19);
-    context.map().unobscuredCenterZoomEase(extent.center(), setZoom);
+    const graph = context.graph();
+    let selectID;
+    let focusCenter;
+
+    // Try to focus the map at the center of the issue..
+    const issueExtent = issue.extent(graph);
+    if (issueExtent) {
+      focusCenter = issueExtent.center();
+    }
 
-    // select the first entity
+    // Try to select the first entity in the issue..
```

Comment:
I did consider that, but I think because relations can be very large it's ok to just find the first reasonable node or way, rather than looking at all of them to decide which one is closest (which could take a long time).